launcher/ui/dialogs/BlockedModsDialog.h

```diff
@@ -79,6 +79,7 @@ class BlockedModsDialog : public QDialog {
     void update();
     void directoryChanged(QString path);
     void setupWatch();
+    void watchPath(QString path, bool watch_subdirectories = false);
```

Comment:
Maybe call the parameter `recursive` instead. The current name could also mean that it doesn't descend more than once.

Reply:
fixed :+1: